chopper/lib/src/base.dart

```diff
@@ -349,9 +343,9 @@ class ChopperClient {
   /// Makes a HTTP GET request using the [send] function.
   Future<Response<BodyType>> get<BodyType, InnerType>(
     String url, {
-    Map<String, String> headers,
-    Map<String, dynamic> parameters,
-    String baseUrl,
+    Map<String, String>? headers,
```

Comment:
To avoid uneccessary null value

All `List` and `Map` could be initialize to empty List/Map

Reply:
Fixed. Please check